Find a functionally equivalent code:android.os.Bundle.putByteArray:COMMENT
Method Modifier: public      
Comment:/**
 * Inserts a byte array value into the mapping of this Bundle, replacing
 * any existing value for the given key.  Either key or value may be null.
 *
 * @param key a String, or null
 * @param value a byte array object, or null
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    mMap.put(key, value);
+    super.putByteArray(key, value);
 }

